app/views/propertyrepresentation/appoint/appointAgentProperties.scala.html

```diff
@@ -65,19 +65,15 @@
 @addressColumnHeaderLink = {
     <a id="sort-by-address"
        href="@agentAppointment.routes.AppointAgentController.sortPropertiesForAppoint(ADDRESS, pagination, agentCode, agentAppointed = appointedAgentToggle, backLinkAsString).url"
-       class="govuk-link"
-       role="link"
-       style="color: black">
+       role="link">
```

Comment:
do we not need `class="govuk-link"`?
do we need/use `role="link"`?

Reply:
We don't need class="govuk-link" for sortable header columns as styles for sortable headers declared in _sortable.scss

I kept role="link" as prototype and other places declared this attribute, may be used for other purpose but not sure where it is being used